torch/csrc/api/include/torch/functional.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <torch/types.h>
+
+namespace torch {
+
+inline at::Tensor isfinite(const at::Tensor& tensor) {
```

Comment:
Do we still need this file? I thought that all aten functions get bound into the torch namespace, so `torch::isfinite` might exist already

Reply:
deleted